dhis2/dhis2-core
# feat: login redirect pattern check

### Summary
This prevents a redirection loop in the embedded Tomcat and Jetty server, which would result in redirect after log in to always redirect to / which would then be interpreted as default app the dashboard.

### Manual test

1. Start server with embedded Tomcat or Jetty 
2. Try to enter app URL directly (before logged in), e.g. : http://localhost:8080/dhis-web-data-visualizer/index.html
3. Log in
4. Observe you end up on the URL entered in step 2

### Automatic
Not feasible right now, since E2E is not using embedded servers.
